src/store/mutations.js

```diff
@@ -96,4 +97,30 @@ export default {
   setLoginTargetLocation(state, location) {
     state.loginTargetLocation = location;
   },
+  createAccount(state) {
+    state.accs.push({
+      idx: state.accountCount,
+      userName: i18n.t('subaccountName', { number: state.accountCount }),
```

Comment:
More accurate would be to add this in `accounts` getter (`userName || i18n.t(...)`) to keep it translatable and be able to change the wording in the future.